bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/net/NetUtil.java

```diff
@@ -21,14 +21,21 @@
 import java.net.UnknownHostException;
 import java.util.ArrayList;
 import java.util.Collection;
+import java.util.Collections;
 import java.util.Enumeration;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.TimeUnit;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
 import org.eclipse.jdt.annotation.NonNullByDefault;
 import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.core.scheduler.ExpressionThreadPoolManager;
+import org.eclipse.smarthome.core.scheduler.ExpressionThreadPoolManager.ExpressionThreadPoolExecutor;
```

Comment:
Damn I was too quick... Since I saw your change that you use `schedulewithfixeddelay` now I thought all is good. But since you do not have the CronExpression anymore you can replace this `ExpressionThreadPoolManager` with `ScheduledExecutorService scheduler = ThreadPoolManager.getScheduledPool(THING_HANDLER_THREADPOOL_NAME);`

Reply:
Thanks for suggestion, replaced